autoload/gitgutter/hunk.vim

```diff
@@ -91,18 +99,18 @@ function! gitgutter#hunk#prev_hunk(count) abort
       let hunk_count += 1
       if hunk_count == a:count
         let target = hunk[2] == 0 ? 1 : hunk[2]
```

Comment:
When changing this I didn't quite understand why this is only necessary for forward jumping. 
Can this special case also happen when jumping tot the next hunk?

If so, i can update it to check it in the `gitgutter#hunk#go_to)` as well. I think some more overall simplification can then be done.